src/modules/ipfs/index.ts

```diff
@@ -0,0 +1,8 @@
+
+export const fromCloudFlare = (cid: string) => (
+  `https://cloudflare-ipfs.com/ipfs/${cid}`
+)
+
+export const fromDwebLink = (cid: string) => (
+  `https://${cid}.ipfs.dweb.link`
+)
```

Comment:
```suggestion
export const fromCloudFlare = (cid: string):string => (
  `https://cloudflare-ipfs.com/ipfs/${cid}`
)

export const fromDwebLink = (cid: string):string => (
  `https://${cid}.ipfs.dweb.link`
)
```

The return type may be inferred but if not.